pages/Documents/ConversationalAI/Third-PartyBots/conversation-tester.md

```diff
@@ -28,6 +28,28 @@ When testing your bot and you get an error in the response, you will be able to
 ## Dialog Flow V2 Invalid Action Body
 ## Description
 This error happens when the bot has an invalid body and the Google Dialog Flow V2 is unable to parse it. It can be a wrong parameter, a value, or a missing parameter. Even though it is shown as an error in the Third Party Bots, the bot gently ignores unknown actions and the conversation will continue.
+
+## Transfer Action Error
+## Description
+This error happens when either one or multiple of the following conditions not met
+- `skill` is the required parameter and it has to be provided as a **String**
+- `agentId` is an optional parameter and if provided the value must be a **String**
+
+## Change TTR Action Error 
+## Description
+This error happens when the required parameter `ttrType` is either not provided or if provided the format of the value is not a **String**
+
+
+## Close Conversation Action Error
+## Description
+For this action there are no required parameters. When the withoutPcs optional parameter is provided the format should be **Boolean**
+
+## Faas Invocation Action Error
+## Description
+This error happens when either one or multiple of the following conditions not met
+- `lambdaUuid` has not being provided as a **String** 
```

Comment:
also identify which of them is optional or required

Reply:
done